fv3_cap.F90

```diff
@@ -74,7 +75,7 @@ module fv3gfs_cap_mod
 
   type(ESMF_GridComp)                         :: fcstComp
   type(ESMF_State)                            :: fcstState
-  character(len=80),         allocatable      :: fcstItemNameList(:)
+  character(len=max_filename_len+14),allocatable :: fcstItemNameList(:)
```

Comment:
How is the "14" defined? This could be confusing and error-prone when the interpolation method is changed, such as "first_order_conservative". It seems a temporary fix, from that point of view, I would suggest just setting the length of all the character strings to "ESMF_MAXSTR". Not sure the reason to have the filenames exactly 255 characters. If we do want that, we need a different fix. Basically the filenames are the local file names, if developers want to write to a different directory, they can set that up in script level.

Reply:
agreed and done.